apps/rush-lib/src/logic/PublishUtilities.ts

```diff
@@ -127,7 +127,9 @@ export class PublishUtilities {
   public static sortChangeRequests(allChanges: IChangeInfoHash): IChangeInfo[] {
     return Object.keys(allChanges)
       .map((key) => allChanges[key])
-      .sort((a, b) => (a.order! < b.order! ? -1 : 1));
+      .sort((a, b) =>
+        a.order! === b.order! ? a.packageName.localeCompare(b.packageName) : a.order! < b.order! ? -1 : 1
```

Comment:
@D4N14L I noticed you pushed this after I approved the PR.  FYI this can probably be done easier using the [Sort](https://rushstack.io/pages/api/node-core-library.sort/) API.